sw/host/opentitantool/config/opentitan.json

```diff
@@ -2,23 +2,69 @@
   "pins": [
     {
       "name": "RESET",
-      "mode": "OpenDrain",
+      "mode": "PushPull",
```

Comment:
@cfrantz, I changed this to PushPull because OpenDrain is not supported on cw310 and I wasn't successful in overriding it both here in "strappings" and opentitan_cw310.json.

Reply:
That will need some investigation.  I'd suggest leaving a comment, but comments are forbidden in json files. :)

Could you file an issue to investigate the use cases and flexibility of the opentitantool config files?

Reply:
Created #12828